public/js/all.js

```diff
@@ -372,22 +372,6 @@ jQuery(document).ready(function onReady($) {
     $('.searchusericon').attr('src', '/UserPic/' + ui.item.id + '.png');
     return false;
   });
-
-  var $resetForm = $('#resetform');
```

Comment:
Given the age of this codebase, still using jquery (for the better), we somehow happen to use javascript in a quite inflationary fashion. Simple HTML forms are much easier to reason about and easier to respond to in the backend. Let's get rid of that if possible. There's no "ajax fanciness" to be gained here.